framework/test/jest/unit/specs/modules/chain/synchronizer/block_synchronization_mechanism/block_synchronization_mechanism.spec.js

```diff
@@ -247,5 +257,121 @@ describe('block_synchronization_mechanism', () => {
 				).toEqual([input[1], input[2]]);
 			});
 		});
+
+		describe('async run()', () => {
+			it('when there are no errors', async () => {
```

Comment:
probably this should another `describe`, and `it` should describe what it is checking.
Same for below

Reply:
Done.